Added bootstrap collapse component

aharnisch opened the pull request:
Implementation of Bootstrap Collapse component: https://getbootstrap.com/docs/5.3/components/collapse/


Test code:
```kotlin
val id1 = "collapseExample"
val id2 = "collapseExample2"
val group = "collapseGroup"
buttonGroup {
	button("Toggle #1") {
		forCollapse(id1)
	}
	button("Toggle #2") {
		forCollapse(id2)
	}
	button("Toggle both") {
		forCollapse(group, id1, id2)
	}
}
div(className = "row") {
	this.width = 200.px
	div(className = "col") {
		collapse(id1, group, opened = true) {
			+"Hello Collapse 1"
		}
	}

	div(className = "col") {
		collapse(id2, group) {
			+"Hello Collapse 2"
		}
	}
}
```

Hi Robert,
I would like to contribute to your project. Please give me feedback to improve my pull requests to match your requirements.

rjaros commented:
I've made some small fixes and added a test class. Take a look at my commits for guidelines. It's now merged to the master branch. Thank you very much for your contribution :-)

aharnisch commented:
If I see it correctly you updated the copyright block, added few Suppress annotations and changed the scope of the init functions..

I will work on Accordion and Badge next.

rjaros commented:
You are correct. The suppress annotations were needed because I've made the class `open`.
